paperoni/webapp/__init__.py

```diff
@@ -1,12 +1,23 @@
+import logging
+
 from .common import config
 
 cfg = config()
 
 if hasattr(cfg, "sentry") and cfg.sentry.use:
     import sentry_sdk
+    # Configure sentry to collect log events with minimal level INFO
+    # (2023/10/25) https://docs.sentry.io/platforms/python/integrations/logging/
+    from sentry_sdk.integrations.logging import LoggingIntegration
 
     sentry_sdk.init(
         dsn=cfg.sentry.dsn,
         traces_sample_rate=cfg.sentry.traces_sample_rate,
         environment=cfg.sentry.environment,
+        integrations=[
+            LoggingIntegration(
+                level=logging.INFO,
+                event_level=logging.INFO
+            )
```

Comment:
Sorry for the delay! I think this should be in the config file, e.g. `cfg.sentry.log_level` and `cfg.sentry.event_log_level`, so we can change it easily if it turns out to be a bad idea.

Also, I'm moving the sentry code to grizzlaxy so it can be reused across projects that use it, so you could make a PR over there as well: https://github.com/breuleux/grizzlaxy/blob/master/grizzlaxy/cli.py#L108

Reply:
@notoraptor The link above won't work because of new commits, see here instead: https://github.com/breuleux/grizzlaxy/blob/master/grizzlaxy/cli.py#L186